HeterogeneousCore/CUDAServices/scripts/cudaPreallocate.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
```

Comment:
@makortel is this still relevant ?
I think the CUDAService does not support configuring the allocators any more, does it ?

Reply:
> I think the CUDAService does not support configuring the allocators any more, does it ?

Correct, but the preallocation is still possible (it doesn't require configuring the allocators, just using them).

> is this still relevant ?

Possibly not. I wouldn't mind removing the script, but then I'd also remove the corresponding feature from CUDAService.

Reply:
Ah, I see... so the `CUDAService` does still support preallocation.